split.py

```diff
@@ -10,15 +11,18 @@
 # Load processed data
 all_spectro = utils.load_dump(config.EMOTIFY_SPECTROGRAM_DUMP_PATH)
 
-labels = utils.load_labels(config.EMOTIFY_LABELS_DUMP_PATH)
+index = np.array([[i+1] for i in range(len(all_spectro))])
+
+labels = utils.load_dump(config.EMOTIFY_LABELS_DUMP_PATH)
 
 # Train/test split 80/20, stratify on highest ranked emotion
-train_x, test_x, train_df_y, test_df_y = train_test_split(
-    all_spectro, labels, train_size=config.TRAIN_SIZE, random_state=42)
+train_x_index, train_y, test_x_index, test_y = iterative_train_test_split(
```

Comment:
Beware that now train_y and test_y are numpy array and not lists. This might not make a difference but it is a change from what we had before.